src/main/resources/db/migration/company_schema.sql

```diff
@@ -0,0 +1,6 @@
+create table department
```

Comment:
1. Remove the database schema
2. Start the service (application)
3. The expected result: schema should be generated by Flyway